pkg/services/publicdashboards/api/api.go

```diff
@@ -25,6 +25,7 @@ import (
 
 type Api struct {
 	PublicDashboardService publicdashboards.Service
+	DashboardService       dashboards.DashboardService
```

Comment:
Do we actually need the dashboard service for anything?

Reply:
No we dont, thanks!